docker/build.run.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+check_vols_src() {
+  if [ ! -d /vols/src ]; then
+    echo "No /vols/src with code"
+    exit 1
+  fi
+}
+check_vols_out() {
+  if [ ! -d /vols/out ]; then
+    echo "No /vols/out for output!"
+    exit 1
+  fi
+}
+
+function sync {
+  check_vols_src
+  {
+    echo "- .git"
+    echo "- .bin"
+    echo "- node_modules"
+    echo "- tmp"
+  } > /tmp/rsync_exclude
+  rsync -ar --exclude-from=/tmp/rsync_exclude --delete-during /vols/src/ ./
+}
+
+build() {
+  npm install
+  gulp transifex-download
+  gulp build
+  cp ./server/rewrite.htaccess ./server/www/
```

Comment:
FYI there is a `gulp release` command we used before to build a package. I'm not sure if thats useful here or if we should drop it now?
Run like `gulp release --version-suffix=${TRAVIS_TAG:-$TRAVIS_COMMIT}` to append the tag or commitid to the built zip